src/types/gql.ts

```diff
@@ -0,0 +1,13 @@
+import { GraphQLError } from "graphql";
+
+export type ApolloMock<Data, Variables> = {
+  request: {
+    query: any;
```

Comment:
although we type the query as `any` in `react-app-env.d.ts`, I'm thinking that this might be more exact:

```suggestion
    query: DocumentNode;
```
where `DocumentNode` can be imported from the `"graphql"` package. At the very least, I don't think it will break the tests :]